python/hail/methods/statgen.py

```diff
@@ -1298,8 +1299,36 @@ def hwe_normalized_pca(dataset, k=10, compute_loadings=False, as_array=False):
 
     Notes
     -----
-    Variants that are all homozygous reference or all homozygous alternate
-    are removed before evaluation.
+
+    This is a specialization of :meth:`.pca` above for the common use case
+    of PCA in statistical genetics, that of projecting samples to a small
+    number of ancestry coordinates. Variants that are all homozygous reference
+    or all homozygous alternate are unnormalizable and removed before
+    evaluation.
+
+    Users of PLINK/GCTA should be aware that Hail computes the GRM slightly
+    differently with regard to missing data. In Hail, the
+    :math:`ij` entry of the GRM :math:`MM^T` is simply the dot product of rows
+    :math:`i` and :math:`j` of :math:`M`; in terms of :math:`C` it is
+
+    .. math:
+      \\frac{1}{m}\sum_{l\in\mathcal{C}_i\cap\mathcal{C}_j}\\frac{(C_{il}-2p_l)(C_{jl} - 2p_l)}{2p_l(1-p_l)
```

Comment:
Any reason you're not using a raw string?

Reply:
going by the "if it ain't broke and there are more critical needs, don't fix it" philosophy